### DISCUSSION TITLE
Support Programmaing Language emojis

### DISCUSSION THREAD

**Discussion by tanguyantoine** [community/community#32002] [Other Feature Feedback, Questions, & Ideas]
It would be really cool if Github markdown could support programming language emojis. 

```md
# :python: Python
```
woudl generate

# 🐍  Python 

then do it for each programming languages

**Comment by BLKKKBVSIK** [comment]
It would be a nightmare, there are tens of thousands of programming languages, some with the exact same names with different authors.
I think it'll be best if we avoid it because it will be a nightmare to know which ones are still maintained.
